snapd-glib/requests/snapd-json.c

```diff
@@ -68,6 +68,19 @@ _snapd_json_get_string (JsonObject *object, const gchar *name, const gchar *defa
         return default_value;
 }
 
+const gchar *
+_snapd_json_get_int_as_string (JsonObject *object, const gchar *name)
+{
+    JsonNode *node = json_object_get_member (object, name);
+    if (node != NULL) {
```

Comment:
Nit: This uses less indentation if you return if `node == NULL`.

Reply:
Fixed.